frontend/packages/console-dynamic-plugin-sdk/scripts/templates/api.md.ejs

```diff
@@ -23,6 +23,7 @@
 
 <% } %>
 
+<!-- FIXME This section does not render see  https://bugzilla.redhat.com/show_bug.cgi?id=2083154-->
 <% if(api.doc.paramters) { %>
```

Comment:
@TheRealJon I think there's a typo here, should be `<% if (api.doc.parameters) { %>`

Reply:
@vojtechszocs: @TheRealJon fixed it in #11501 👍